pkg/config/cliconfig.go

```diff
@@ -150,11 +159,11 @@ func ReadCLIConfig(gptscriptConfigFile string) (*CLIConfig, error) {
 		errMsg := fmt.Sprintf("invalid credential store '%s'", result.CredentialsStore)
 		switch runtime.GOOS {
 		case "darwin":
-			errMsg += " (use 'osxkeychain' or 'file')"
+			errMsg += " (use 'osxkeychain', 'file', or 'sqlite')"
```

Comment:
These probably won't change much, but you could use `strings.Join(darwinHelpers, ", ")` here for simplicity.

Reply:
Fixed